### PR TITLE
Feature fix doxygen for master

### PR DESCRIPTION
**Pull request by Lennart99**
fixed the doxygen as requested for the PR to master

### CODE REVIEW

**Review by LRstudentHU — APPROVED** [review]

File: code/headers/can_bus_pending_task.hpp [hunk 1]
```diff
@@ -3,13 +3,18 @@
...
+    // task when there are frames ready to be processed
     template <size_t Size>
     class can_bus_pending_task_c : public rtos::task<256> {
     private:
         std::array<module_task_base_c *, Size> tasks;
         size_t next_task_index = 0;
 
     public:
+        // adds a module to the list of tasks
+        // this task will loop thru this list to check for frames for every
```
Comment (denizlektemur) [inline code/headers/can_bus_pending_task.hpp]: thru should be through

File: code/headers/module_task.hpp [hunk 2]
```diff
@@ -23,6 +23,12 @@ namespace r2d2::module_scheduler {
         arena_alloc_c<WaitableAllocSize> allocator;
 
     public:
+        // initializes the module and it's task
```
Comment (denizlektemur) [inline code/headers/module_task.hpp]: it's should be its

**Review by denizlektemur — APPROVED** [review]
Looks good besides the 2 spelling mistakes

File: code/headers/module_task.hpp (hunk 2 above)
Comment (Lennart99) [reply code/headers/module_task.hpp]: https://github.com/R2D2-2019/module_scheduler/pull/5/commits/fbc15c836927b0f79866df015a2da0bd959d0550

File: code/headers/can_bus_pending_task.hpp (hunk 1 above)
Comment (Lennart99) [reply code/headers/can_bus_pending_task.hpp]: https://github.com/R2D2-2019/module_scheduler/pull/5/commits/fbc15c836927b0f79866df015a2da0bd959d0550

**Pull request merged**